pkg/generic_advdiff/GAD_OPTIONS.h

```diff
@@ -19,18 +19,16 @@ C Use this file for selecting options within the GAD package
 #ifdef ALLOW_GENERIC_ADVDIFF
 C     Package-specific Options & Macros go here
 
-C This flag selects the form of COSINE(lat) scaling of bi-harmonic term.
-C *only for use on a lat-lon grid*
-C Setting this flag here only affects the bi-harmonic tracer terms; to
-C use COSINEMETH_III in the momentum equations set it CPP_OPTIONS.h
+C This flag selects the form of COSINE(lat) scaling of horizontal
+C bi-harmonic diffusivity. *only for use on a lat-lon grid*
+C Setting this flag here only affects tracer diffusivity; to use it
+C in the momentum equations it needs to be set in MOM_COMMON_OPTIONS.h
 #define COSINEMETH_III
 
-C This selects isotropic scaling of harmonic and bi-harmonic term when
-C using the COSINE(lat) scaling.
-C Setting this flag here only affects the tracer diffusion terms; to
-C use ISOTROPIC_COS_SCALING of the horizontal viscosity terms in the
-C momentum equations set it CPP_OPTIONS.h; the following line
-C even overrides setting the flag in CPP_OPTIONS.h
+C This selects isotropic scaling of horizontal harmonic and bi-harmonic
+C diffusivity when using the COSINE(lat) scaling. *only for lat-lon grid*
+C Setting this flag here only affects tracer diffusivity; to use it
+C in the momentum equations it needs to be set in MOM_COMMON_OPTIONS.h
```

Comment:
similar to previous comment/suggestion
```suggestion
C *only for lat-lon grid*
C This selects isotropic scaling of horizontal harmonic and bi-harmonic
C diffusivity when using the COSINE(lat) scaling. 
C Setting this flag here only affects tracer diffusivity; to use it
C in the momentum equations it needs to be set in MOM_COMMON_OPTIONS.h
```

Reply:
I like when the restriction "only for lat-lon grid" is part of the main description, on same line, But thinking of this,
I would rather keep it on same line but change it to "-- only on lat-lon grid." (without "*").